packages/core/src/components/external/Stopwatch/Stopwatch.tsx

```diff
@@ -143,55 +168,49 @@ const Stopwatch = ({
       <View style={styles.row}>
         <View style={styles.btnContainer}>
           {stopwatchStatus !== StopwatchType.status.InProgress && (
-            <Icon
+            <TimerButton
               name="play"
-              size={25}
-              disabled={
-                disable ||
-                disablePlay ||
-                stopwatchStatus === StopwatchType.status.InProgress ||
-                stopwatchStatus === StopwatchType.status.Finished
+              touchable={
+                !(
+                  disable ||
+                  disablePlay ||
+                  stopwatchStatus === StopwatchType.status.InProgress ||
+                  stopwatchStatus === StopwatchType.status.Finished
+                )
```

Comment:
A props disabled instead of touchable in TimerButton would simplify those condition